GitUI/BranchTreePanel/RepoObjectsTree.Nodes.Branches.cs

```diff
@@ -53,7 +53,7 @@ protected BaseBranchNode(Tree tree, string fullPath, bool visible)
             /// <summary>
             /// Full path of the branch. <example>"issues/issue1344"</example>.
             /// </summary>
-            public string FullPath => ParentPath.Combine(PathSeparator.ToString(), Name);
+            public string FullPath => ParentPath.Combine(PathSeparator.ToString(), Name)!;
```

Comment:
This `Combine` extension method (it is not `System.IO.Path.Combine`) has pretty weird behaviour and cannot be annotated. It should probably be replaced with something less surprising and easier to annotate. I've suppressed the warning here with `!`.